### PR TITLE
Migration to DynamoDB

### PR DESCRIPTION
**Pull request by Boscoand**
In this code, we have migrate to Dynamo DB from CassandraDB. Also, we have include Dynamoose that is a ORM for Dynamo.

### CODE REVIEW

File: src/config/configDB.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import dotenv from "dotenv"
+
+var dynamoose = require('dynamoose')
+
+dotenv.config()
+
+dynamoose.AWS.config.update({
+  accessKeyId: process.env.AK,
+  secretAccessKey: process.env.SK,
+  region: 'us-east-1'
```
Comment (bitcod3r) [inline src/config/configDB.js]: Please, change this parameter "region" to .env private file.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const authbyrole = (role) => passport.authenticate(role, { session: false })
+
+// parser
+app.use(bodyParser.json());
+// routes
+app.use("/api/auth", auth);
+app.use("/api/private/restaurant", authbyrole(process.env.SA_ROLE), restaurant_private)
+app.use("/api/public/restaurant", restaurant_public)
+
+app.listen(process.env.APP_PORT, () => console.log("Running on localhost: " + process.env.APP_PORT))
```
Comment (bitcod3r) [inline src/index.js]: Please, add the app version number in this console.log.

File: src/routes/auth.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+        errors.errorIncorrectPassword(res)
+
+    })
+
+})
+
+export default router;
+
+/*router.post('/register', (req, res) => {
```
Comment (bitcod3r) [inline src/routes/auth.js]: Refactor the registration methods to use Dynamo.

File: src/errors/errors.auth.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    errorDataBaseConnection: (res) => {
+        res.status(400).json({errors: { global: "Error from database connection." } })
```
Comment (bitcod3r) [inline src/errors/errors.auth.js]: Print in console the error details.

**Review by bitcod3r — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (Boscoand) [reply src/index.js]: Which is the first version? 0.0.1?

File: src/errors/errors.auth.js (hunk 4 above)
Comment (bitcod3r) [reply src/errors/errors.auth.js]: good job!

**Review by bitcod3r — APPROVED** [review]

**Pull request merged**